packages/template-retail-react-app/package.json

```diff
@@ -1,95 +1,95 @@
 {
-    "name": "retail-react-app",
-    "version": "2.7.2",
-    "license": "See license in LICENSE",
-    "private": true,
-    "engines": {
-        "node": "^14.0.0 || ^16.0.0",
-        "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+  "name": "retail-react-app",
+  "version": "2.7.2",
+  "license": "See license in LICENSE",
+  "private": true,
+  "engines": {
+    "node": "^14.0.0 || ^16.0.0 || ^18.0.0",
+    "npm": "^6.14.4 || ^7.0.0 || ^8.0.0 || ^9.0.0"
+  },
+  "devDependencies": {
+    "@chakra-ui/icons": "^1.1.7",
+    "@chakra-ui/react": "^1.8.9",
+    "@chakra-ui/skip-nav": "^1.2.6",
+    "@chakra-ui/system": "^1.12.1",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@formatjs/cli": "^4.8.4",
+    "@lhci/cli": "^0.12.0",
```

Comment:
Version bump for lhci here fixes an issue where lighthouse fails to connect to chrome on Node 18.16
https://github.com/ottofeller/templates/pull/269